packages/editor-ui/src/components/InputPanel.vue

```diff
@@ -186,7 +186,7 @@ export default defineComponent({
 			const workflow = this.workflow;
 			const parentNodes = [...workflow.getParentNodes(this.rootNode, NodeConnectionType.Main)]
 				.reverse()
-				.map((parent): IConnectedNode => ({ name: parent, depth: 1, indicies: [] }));
```

Comment:
Hardcoding the depth of all parent nodes to 1 here broke the generation of expressions when dragging and dropping fields here: https://github.com/n8n-io/n8n/blob/7e44cd7f16e1e8a61f79ccd71705534f6341ef56/packages/editor-ui/src/utils/mappingUtils.ts#L39